.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+# This is a basic workflow
```

Comment:
@MariaMaximova, please move this file to another branch that is dedicated to implementing release github action feature. It is best practive not to leave commended code that does not work yet. So not to lost it you can create new branch from `develop` and put this file there as uncommented and work with new feature separately from this MR.

Reply:
Done

Reply:
[File](https://github.com/MariaMaximova/convert_temp/pull/19/commits/783a509aa1c26b0826ad164b19370b38f1a70377#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34) was deleted